### ISSUE TITLE
Boxstarter hangs after installing Windows Updates

### ISSUE THREAD

**Issue by PilotBob** [chocolatey/boxstarter#150]
This line in my script:

```
# Update Windows and reboot if necessary
Install-WindowsUpdate -AcceptEula

```

runs, it finds a windows defender update, installs it, the console says "Finished..." but it just sits there and doesn't move on. I waited a good 15 minutes and Task Manager showed 0% CPU usage for the process.

**Comment by mwrock** [comment]
mind adding a gist of the boxstarter log? in `$env:localappdata\boxstarter`

**Comment by PilotBob** [comment]
m'kay... ran it again to get a log, and it worked. Of course.

**Comment by mwrock** [comment]
of course :)

**Comment by PilotBob** [comment]
and now chocolatey is down, so I'm on hold. This is supposed to save me time. ;)

**Comment by gep13** [comment]
@PilotBob Chocolatey.org is back up.

**Comment by PilotBob** [comment]
yep, thanks.

**Comment by pauby** [comment]
No updates to this in some time and it's waiting on user response so going to go ahead and close this. If we need to reopen again in the future we can.
